back/spec/services/survey_results_generator_service_spec.rb

```diff
@@ -600,30 +600,59 @@
       end
 
       context 'with grouping' do
-        let(:grouped_linear_scale_answers) do
-          [
-            { answer: 5, count: 1, groups: [
-              { count: 1, group: nil }
-            ] },
-            { answer: 4, count: 1, groups: [
-              { count: 1, group: 'la' }
-            ] },
-            { answer: 3, count: 8, groups: [
-              { count: 1, group: 'ny' },
-              { count: 7, group: nil }
-            ] },
-            { answer: 2, count: 5, groups: [
-              { count: 5, group: nil }
-            ] },
-            { answer: 1, count: 2, groups: [
-              { count: 2, group: nil }
-            ] },
-            { answer: nil, count: 5, groups: [
-              { count: 1, group: 'la' },
-              { count: 3, group: 'other' },
-              { count: 1, group: nil }
-            ] }
-          ]
+        let(:grouped_linear_scale_results) do
```

Comment:
Grouped linear scales were actually not working correct, which I didn't know because I was lazy when writing tests (was only testing the answers, not the whole response). Now the test is correct

Reply:
What exactly was broken? Maybe we could include only the fragile parts that need to be tested? A bit similarly as I did here by using `include` instead of `match` https://github.com/CitizenLabDotCo/citizenlab/pull/7375/files#diff-37ad47fa7b36779df9339d5b66fc05271abebb25d50bca0fe1fdd0dc71c3dd17R41

This way, the test becomes 
1. shorter and easier to read by highlighting the important testing parts
2. less brittle. E.g., when copies or locales change.